### PR TITLE
gh-118 toError cover with unit-tests

### PR DESCRIPTION
**Pull request by oleksii-yurchenko**
- extract toError func
- Make basic happy path tests

### CODE REVIEW

File: packages/util-use-case/lib/adapters/toError.test.mjs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  t.assert(
+    res.send.calledOnceWith({
+      status: 0,
+      error: error.toJSON(),
+    }),
+    'Should send this object if error instance of ValidationError'
+  )
+
+  // t.assert(deps.log instanceof MockLogger, 'Deps should have prop log whitch istanceof MockLogger')
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toError.test.mjs]: add here  checks that log.fatal was not called

File: packages/util-use-case/lib/adapters/toError.mjs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import {MockLogger} from '@zorko-io/util-logger'
+import {ValidationError} from '@zorko-io/util-error'
+
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toError.mjs:3]: Add [jsdocs](https://jsdoc.app/) notation, as an example refer to `makeRunner`

**Review by nesterone — CHANGES_REQUESTED** [review]

File: packages/util-use-case/lib/adapters/toError.mjs (hunk 2 above)
Comment (oleksii-yurchenko) [reply packages/util-use-case/lib/adapters/toError.mjs:3]: Do you mean smth like that?
/**
 * Passes result to response as json,
 * @param {Error}error - error to convert
 * @param {Express.Request} req - request
 * @param {Express.Response} res - response
 * @param {Object} [deps] - dependencies
 * @param {CoreLogger} [deps.log] - logger instance
 * @return {*} - converted result, the same what we send over response
 */

Comment (nesterone) [reply packages/util-use-case/lib/adapters/toError.mjs:3]: yep
```
/**

Convert error to JSON and pass to response
@param {Error}error - error to convert
@param {Express.Request} req - request
@param {Express.Response} res - response
@param {Object} [deps] - dependencies
@param {CoreLogger} [deps.log] - logger instance
@return {*} - converted result, the same what we send over response
*/


```

**Review by nesterone — APPROVED** [review]

**Pull request merged**